tools/sidekick/plugins.js

```diff
@@ -17,13 +17,70 @@
 
   // sk.loadCSS();
 
+  // EDIT -------------------------------------------------------------------------
+
+  sk.add({
+    id: 'edit',
+    condition: (sidekick) => sidekick.isHelix(),
+    override: true,
+    button: {
+      action: () => {
+        const { config, location } = sk;
+        const href = location.href
```

Comment:
why not using the URL class to extract the needed pieces ?

Reply:
Hmm. You mean something like this?
```js
const url = new URL(location.href);
url.hash = '';
url.search = '';
url.pathname = url.pathname.replace(/\/([a-z]{2})\/(\d{4})/, '/$1/publish/$2');
const href = url.toString();
```
I thought about it but considered it to be clunkier. But maybe it's safer...

Reply:
We can still change it if it breaks.